engine/src/engine.ts

```diff
@@ -109,7 +109,7 @@ export default class Engine {
     federations: {},
   };
   boardActions: {
-    [key in BoardAction]?: boolean;
+    [key in BoardAction]?: PlayerEnum;
```

Comment:
This is fine, but to keep compatibility with ongoing games, in `Engine.fromData`, if the loaded board actions contain booleans, they should be set to either `null` or `Player.Player5`.